mysql-wordpress-pd/README.md

```diff
@@ -4,367 +4,195 @@
+
+       kubectl get pods
+
+   The response should be similar to this:
+
+       NAME                               READY     STATUS    RESTARTS   AGE
+       wordpress-mysql-1894417608-x5dzt   1/1       Running   0          40s
+
 
 ## Deploy WordPress
 
-Next deploy WordPress using
-[wordpress-deployment.yaml](https://git.k8s.io/examples/mysql-wordpress-pd/wordpress-deployment.yaml):
+The following manifest describes a single-instance WordPress Pod Deployment. It uses many of the same features like a PVC for persistent storage and a Secret for the password. However, it also uses a different setting: `"type": NodePort`. This setting exposes WordPress to traffic from outside of the cluster.
+
+{% include code.html language="yaml" file="mysql-deployment.yaml" ghlink="/docs/tutorials/stateful-application/wordpress-deployment.yaml" %}
+
+1. Deploy WordPress from the `wordpress-deployment.yaml` file:
+
+       kubectl create -f wordpress-deployment.yaml
+
+2. Verify that the Service is running by running the following command:
 
-```shell
-kubectl create -f $KUBE_REPO/mysql-wordpress-pd/wordpress-deployment.yaml
-```
+       kubectl get services wordpress
 
-Here we are using many of the same features, such as a volume claim
-for persistent storage and a secret for the password.
+   The response should be similar to this:
 
-The [WordPress image](https://hub.docker.com/_/wordpress/) accepts the
-database hostname through the environment variable
-`WORDPRESS_DB_HOST`. We set the env value to the name of the MySQL
-service we created: `wordpress-mysql`.
+       NAME        CLUSTER-IP   EXTERNAL-IP   PORT(S)        AGE
+       wordpress   10.0.0.89    <pending>     80:32406/TCP   4m
 
-The WordPress service has the setting `type: LoadBalancer`.  This will
-set up the wordpress service behind an external IP.
+   **Note:** If you have a cluster running on a Cloud Provider, you can use `"type": LoadBalancer` to automatically expose a Service behind an external IP Address. However, Minikube can only expose Services through `NodePort`. <br/><br/>The `EXTERNAL-IP` is always `<pending>` when in Minikube.
+   {: .note}
 
-Find the external IP for your WordPress service. **It may take a minute
-to have an external IP assigned to the service, depending on your
-cluster environment.**
+3. Run the following command to get the IP Address for the WordPress Service:
 
-```shell
-kubectl get services wordpress
-```
+       minikube service wordpress --url
 
-```
-NAME        CLUSTER-IP     EXTERNAL-IP     PORT(S)   AGE
-wordpress   10.0.0.5       1.2.3.4         80/TCP    19h
-```
+   The response should be similar to this:
 
-## Visit your new WordPress blog
+       http://1.2.3.4:32406
 
-Now, we can visit the running WordPress app. Use the external IP of
-the service that you obtained above.
+4. Copy the IP address, and load the page in your browser to view your blog.
 
-```
-http://<external-ip>
-```
+   You should see the familiar WordPress init page.
```

Comment:
the familiar WordPress init page &rarr; the WordPress set up page, similar to the following screenshot: